daemon/info.go

```diff
@@ -54,9 +54,24 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 	initPath := utils.DockerInitPath("")
 	sysInfo := sysinfo.New(true)
 
+	var cRunning, cPaused, cStopped int
+	for _, c := range daemon.List() {
+		if c.IsRunning() {
```

Comment:
The check for paused and running needs to be switched because paused containers are technically "running" too, which is why my count of paused containers always shows as zero.  Checking for paused first should fix this.

I would suggest that IsRunning() should probably return false for a paused container but based on my scan of the code that is kind of a pervasive change and probably out of scope of this PR and would need a lot more discussion (ie. bikeshedding) first.